README.md

```diff
@@ -6,36 +6,121 @@
 
-### 🚧 UNDER CONSTRUCTION 🚧
+## About
 
 [Zebra](https://zebra.zfnd.org/) is the Zcash Foundation's independent,
 consensus-compatible implementation of the Zcash protocol, currently under
 development.  Please [join us on Discord](https://discord.gg/na6QZNd) if you'd
 like to find out more or get involved!
 
+## Alpha Releases
+
+Our first alpha release is planned for December 8th, 2020.
+
+The goals of this release are to:
+- participate in the Zcash network,
+- replicate the Zcash chain state, and
+- implement the Zcash proof of work consensus rules,
+- sync on Mainnet under excellent network conditions.
+
+Currently, Zebra does not validate all the Zcash consensus rules. It may be
+unreliable on Testnet, and under less-than-perfect network conditions. See
+our [current features](#current-features) and [roadmap](future-work) for
+details.
+
+### Getting Started
+
+Run `cargo ...` **TODO**
+
+If you're interested in testing out `zebrad` please feel free, but keep in mind
+that there is a lot of key functionality still missing.
+
+### System Requirements
+
+**TBD**
+
+### Current Features
+
+Network:
+- synchronize the chain from peers
+- download gossipped blocks from peers
+- answer inbound peer requests for hashes, headers, and blocks
+
+State:
+- persist block, transaction, UTXO, and nullifier indexes
+- handle chain reorganizations
+
+Proof of Work:
+- validate equihash, block difficulty threshold, and difficulty adjustment
+- validate transaction merkle roots
+
+Validating proof of work increases the cost of creating a consensus split
+between `zebrad` and `zcashd`.
+
+This release also implements some other Zcash consensus rules, to check that
+Zebra's [validation architecture](#architecture) supports future work on a
+full validating node:
+- block and transaction structure
+- checkpoint-based verification up to Sapling
+- transaction validation (incomplete)
+- transaction cryptography (incomplete)
+- transaction scripts (incomplete)
+- batch verification (incomplete)
+
+### Future Work
+
+In 2021, we intend to add RPC support and wallet integration.  This phased
+approach allows us to test Zebra's independent implementation of the consensus
+rules, before asking users to entrust it with their funds.
+
+Features:
+- full consensus rule validation
+- transaction mempool
+- wallet functionality
+- RPC functionality
+
+Performance and Reliability:
+- reliable syncing on Testnet
+- reliable syncing under poor network conditions
+- batch verification
+- performance tuning
+
+### Known Issues
+
+There are a few bugs in the Zebra alpha release that we're still working on
+fixing:
+- [Occasional panics in the `tokio` time wheel implementation #1452](https://github.com/ZcashFoundation/zebra/issues/1452)
+- [Peer connections sometimes fail permanently #1435](https://github.com/ZcashFoundation/zebra/issues/1435)
+  - these permanent failures can happen after a network disconnection, sleep, or individual peer disconnections
+
```

Comment:
- [Duplicate block errors #1372](https://github.com/ZcashFoundation/zebra/issues/1372)
    - these errors can be ignored, unless they happen frequently